dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -238,21 +218,6 @@ public void start(WorkerNetAddress address) throws IOException {
     mAddress = address;
     register();
     mOpenFileHandleContainer.start();
-
-    // setup worker-master heartbeat
-    // the heartbeat is only used to notify the aliveness of this worker, so that clients
-    // can get the latest worker list from master.
-    // TODO(bowen): once we set up a worker discovery service in place of master, remove this
-    // TODO(lucy): temporary fallback logic during transition of removing master dependency
-    if (mMembershipManager instanceof NoOpMembershipManager) {
```

Comment:
oh was this removed? I thought @jja725 still needed worker register for a while?